internal/handshake/hkdf.go

```diff
@@ -0,0 +1,29 @@
+package handshake
+
+import (
+	"crypto"
+	"encoding/binary"
+
+	"golang.org/x/crypto/hkdf"
+)
+
+// hkdfExpandLabel HKDF expands a label.
```

Comment:
Why not change qtls's implementation instead?

Reply:
That would be an option. Just harder to test, since I'd like to run it against the standard library HKDF-Expand-Label. Unfortunately, it's not exported though.